zip-0226.rst

```diff
@@ -281,11 +291,17 @@ For every Custom Asset that is burnt, we add to the :math:`\mathsf{assetBurn}` v
 
 .. math:: \mathsf{assetBurn} = \{ (\mathsf{AssetBase},\mathsf{v^{AssetBase}})\ |\ \forall\ \mathsf{AssetBase}\ \textit{s.t.}\ \mathsf{v^{AssetBase}} \neq 0 \}
 
+We denote :math:`|\mathsf{assetBurn}|` the cardinality of the :math:`\mathsf{assetBurn}` set.
+
 The value balances for each Asset Identifier in :math:`\mathsf{assetBurn}` represents the amount of that Asset that is being burnt. In the case of ZEC, the value balance represents either the transaction fee, or the amount of ZEC changing pools (eg: to Sapling or Transparent).
 
-The validator needs to verify the Balance and Binding Signature by adding the value balances for all Assets, as committed using their respective :math:`\mathsf{AssetBase}` as the value base point of the Pedersen Commitment. This is done as follows
+The validator needs to verify the Balance and Binding Signature by adding the value balances for all Assets, as committed using their respective :math:`\mathsf{AssetBase}` as the value base point of the Pedersen Commitment.
+
+This is done as follows:
 
-.. math:: \mathsf{bvk = (\sum cv_i^{net})}  - \mathsf{ ValueCommit_0^{Orchard}(v^{balanceOrchard})} - \sum_{\mathsf{assetBurn}} \mathsf{ValueCommit_0^{OrchardZSA}(AssetBase, v^{AssetBase}) } = \sum \mathsf{rcv_{i,j}^{net}}\mathcal{R}^{\mathsf{Orchard}}
+.. math:: \mathsf{bvk} = \mathsf{(\sum_{i=1}^{n} cv_i^{net})}  - \mathsf{ValueCommit_0^{OrchardZSA}}(\mathsf{v^{balanceOrchard}}, \mathcal{V}^{\mathsf{Orchard}}) - \mathsf{\sum_{k=1}^{|assetBurn|} ValueCommit_0^{OrchardZSA}(v_{k}^{AssetBase}, AssetBase_{k})}
```

Comment:
`\sum_{k=1}^{|assetBurn|}` is quite unusual and looks strange, replace `|assetBurn|` with a letter here and in `We denote :math:`|\mathsf{assetBurn}|` the cardinality of...`

lets remove `^AssetBase` from `(v_{k}^{AssetBase}`. It does not contribute anything.

Reply:
Sure, I can introduce an extra variable for the cardinality (note that `|S|` is quite standard notation for the cardinality though, e.g. https://en.wikipedia.org/wiki/Cardinality)

Reply:
> lets remove ^AssetBase from (v_{k}^{AssetBase}. It does not contribute anything.

Happy to remove these things, I'm just trying to be consistent with the existing notations (v^AssetBase is used throughout this ZIP)